Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM --platform=linux/amd64 golang:1.22-alpine AS builder
+FROM --platform=linux/amd64 golang:1.22-alpine3.20 AS builder
```

Comment:
Is this file still being used?

Reply:
Hmm, I'm guessing it's not since we have a dockerfile just for goreleaser and that's what we use for builds.